README.md

```diff
@@ -54,6 +54,12 @@ module "example" {
 | private_ip | The private IP of the EC2 instance |
 | subnet_id | The ID of the subnet where the EC2 instance is deployed |
 
+## Notes ##
+
+Running `pre-commit` requires running `terraform init` in every directory that
+contains Terraform code. In this repository these are the main directory and the
```

Comment:
There should be a comma after `In this repository`.

Reply:
Fixed in a2ec526.

Reply:
👍